test/integration/main_test.go

```diff
@@ -61,6 +62,15 @@ var (
 	gcsClient *storage.Client
 )
 
+type BQRow struct {
+	Data       string
+	Attributes string
```

Comment:
Does it work that instead of two string here, the type of Data is `pmapevent` and the type of Attributes is the  BQAttributes you defined?

Reply:
Done.